app/src/main/kotlin/com/softteco/template/data/device/ThermometerData.kt

```diff
@@ -0,0 +1,15 @@
+package com.softteco.template.data.device
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class ThermometerData(
+    val deviceId: UUID = UUID.randomUUID(),
```

Comment:
@pcfaktor  left such a comment in my pr: "Can an object of this class be used without deviceId or deviceName? Are default values for these fields really necessary?"